openssl/openssl.rs

```diff
@@ -37,7 +37,7 @@ mod pbkdf2 {
     use test;
 
     pbkdf2_bench!(hmac_sha256, 20, out, {
-        openssl::pkcs5::pbkdf2_hmac(
+        let _ = openssl::pkcs5::pbkdf2_hmac(
```

Comment:
I don't think we should ignore the result. Instead we should panic if the result isn't `Ok` so that we include the cost of checking the result in the benchmark.